recipes/towhee/meta.yaml

```diff
@@ -0,0 +1,124 @@
+{% set name = "towhee" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: bf44c3a7ccce4d57a38b7a34d72095f8142c1d1fd5d3d5c76651a370de034fc8
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - pytorch >=1.4.0
+    - pyyaml >=5.3.0
+    - requests >=2.12.5
+    - tqdm >=4.59.0
```

Comment:
Are all of these really required? Upstream only lists `setuptools_scm`: https://github.com/towhee-io/towhee/blob/43222fc1daf0e2209469580e410c862af4095f5c/setup.py#L44

The pyproject.toml lists more so you might have to just try what the required host dependencies are or ask upstream: https://github.com/towhee-io/towhee/blob/main/pyproject.toml

Reply:
Many thanks for taking time to review, That helps a lot!
I generated this file by `conda skeleton` from pip package, and altered a little bit. I'll check it again.

Reply:
All of them are needed.

Reply:
Interesting, I would have assumed that `setuptools_scm` is required. If not and you have the time, I would suggest making a PR upstream and removing it and any other dependencies that aren't needed, just to keep the packages in sync. 

`conda skeleton` is ok but I would suggest using [Graykull](https://github.com/conda-incubator/grayskull) nowadays. It's much more capable and keeps the recipes along the conda-forge style.